### PR TITLE
added helm repo info and eclipse- prefix in service names

### PR DESCRIPTION
**Pull request by soheiladnde**
currently all service names start with eclipse-, I updated readme accordingly and also added helm repo info for hono helm chart.
Signed-off-by: Soheila Dehghanzadeh <s.dehghanzadeh@denso-auto.de>

### CODE REVIEW

File: charts/hono/README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ kubectl create namespace hono
 The chart can then be installed to name space `hono` using release name `eclipse-hono`:
 
 ```bash
+helm repo add eclipse-iot https://eclipse.org/packages/charts
+helm repo update
```
Comment (sophokles73) [inline charts/hono/README.md]: FMPOV this is already covered by the *Prerequisites* section referring to the website. If we repeat these instructions here we are running at risk of inconsistencies if we change e.g. the repo URL or something. I would therefore rather omit it here ...

File: charts/hono/README.md [hunk 2]
```diff
@@ -63,23 +65,23 @@ Kubernetes *Services*. The following command lists all services and their endpoi
...
+eclipse-hono-artemis                            ClusterIP      10.97.31.154     <none>           5671/TCP
+eclipse-hono-dispatch-router                    ClusterIP      10.98.111.236    <none>           5673/TCP
+eclipse-hono-dispatch-router-ext                LoadBalancer   10.109.220.100   10.109.220.100   15671:30671/TCP,15672:30672/TCP
+eclipse-hono-grafana                            ClusterIP      10.110.61.181    <none>           3000/TCP
+eclipse-hono-prometheus-server                  ClusterIP      10.96.70.135     <none>           9090/TCP
+eclipse-hono-service-auth                       ClusterIP      10.109.97.44     <none>           5671/TCP
+eclipse-hono-service-auth-headless              ClusterIP      None             <none>           <none>
+eclipse-hono-service-device-registry            ClusterIP      10.105.190.233   <none>           5671/TCP
+eclipse-hono-service-device-registry-ext        LoadBalancer   10.101.42.99     10.101.42.99     28080:31080/TCP,28443:31443/TCP
+eclipse-hono-service-device-registry-headless   ClusterIP      None             <none>           <none>
```
Comment (sophokles73) [inline charts/hono/README.md]: nice catch :+1:

**Comment by sophokles73** [comment]
@soheiladnde thanks for contributing :+1: 
you will need to create an Eclipse Foundation account, sign the ECA and then add a "Signed-off-by" footer to your commit messages. See https://www.eclipse.org/packages/contribute/ for details/links

**Comment by soheiladnde** [comment]
I recreated the pull request - #70

**Pull request closed without merging**